### PR TITLE
Adding Automatic-Module-Name to MANIFEST

### PR DESCRIPTION
**Pull request by kiiadi**
So that the API and flow-bridge it can be consumed by Java9 modularized applications.

### CODE REVIEW

File: flow-adapters/build.gradle [hunk 1]
```diff
@@ -8,6 +8,12 @@ dependencies {
 }
 test.useTestNG()
 
+jar {
+    manifest {
+        attributes('Automatic-Module-Name': 'org.reactivestreams.flowbridge')
```
Comment (viktorklang) [inline flow-adapters/build.gradle]: `flowbridge` has been renamed to `flowadapters`

**Comment by viktorklang** [comment]
Thanks @kiiadi!
Does it make sense to add module names to the TCK also?

**Comment by ktoso** [comment]
Though mostly just for the sake of consistency, please add these to all modules (tck, examples)?

**Comment by ktoso** [comment]
Looks good in general though, adding this metadata is very important indeed

**Comment by viktorklang** [comment]
Hi @kiiadi, Would you mind adding the metadata to all the modules? Something like this: https://github.com/reactive-streams/reactive-streams-jvm/commit/26b086b6d5156bd0d2fa314bf819314035013267 ?

**Comment by viktorklang** [comment]
@ktoso Does this make sense from a module-name PoV: https://github.com/reactive-streams/reactive-streams-jvm/commit/26b086b6d5156bd0d2fa314bf819314035013267

**Comment by kiiadi** [comment]
@viktorklang your commit looks fine to me - let's go with that - we can close this PR.

**Comment by ktoso** [comment]
(I commented on the commit, not sure you saw that Victor — all good, would likely drop the api name in the module but having is is still legal)

**Comment by viktorklang** [comment]
@kiiadi You sure? I'd prefer to have you on that commit id :)

**Comment by viktorklang** [comment]
@ktoso I don't mind either way. We can drop the "api" suffix on the primary RS jar

**Comment by viktorklang** [comment]
@reactive-streams/contributors I'll merge this and then add the other module names in a separate PR. Thanks @kiiadi!

**Comment by viktorklang** [comment]
@kiiadi I'll include your commit/pr in this PR here (I resolved the conflict between master/HEAD and your PR as well)

**Pull request closed without merging**